static/js/components/dashboard/CoursePrice.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+import React from 'react';
+import _ from 'lodash';
+
+import type { Course } from '../../flow/programTypes';
+import {
+  STATUS_ENROLLED_NOT_VERIFIED,
+  STATUS_OFFERED_NOT_ENROLLED,
+} from '../../constants';
+import { formatPrice } from '../../util/util';
+
+export default class CoursePrice extends React.Component {
+  props: {
+    course: Course,
+  };
+
+  render() {
+    const { course } = this.props;
+    let firstRun = {};
+    let price = null;
+    let enrollmentStatus = null;
+
+    if (course.runs.length > 0) {
+      firstRun = course.runs[0];
+      if (course.status === STATUS_OFFERED_NOT_ENROLLED || course.status === STATUS_ENROLLED_NOT_VERIFIED) {
+        if (!_.isNil(firstRun.price)) {
+          price = <span className="course-price-display">{formatPrice(firstRun.price)}</span>;
+        }
+        enrollmentStatus = <span className="course-enrollment-description">Enrollment open</span>;
```

Comment:
It looks like there's more to #925 than just showing this text. Can you remove it altogether and it will be added as part of #925?